src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/stores/MemoryFormStore.js

```diff
@@ -2,7 +2,7 @@
 import {action, observable} from 'mobx';
 import type {IObservableValue} from 'mobx';
 import jsonpointer from 'json-pointer';
-import {createAjv} from '../../../utils';
+import {createAjv} from '../../../utils/Ajv';
```

Comment:
@nnatter Do you have an idea why this change fixes this error? https://github.com/sulu/sulu/pull/5815/checks?check_run_id=1880372111

Reply:
because these test classes mock the `sulu-admin-bundle/utils` modules instead of the specific module (for example `sulu-admin-bundle/utils/Translator`) 🙈  
see for example https://github.com/sulu/sulu/blob/466edcb9beb7debe5c64e75bade7d04bbb6164de/src/Sulu/Bundle/MediaBundle/Resources/js/containers/MediaCollection/tests/MediaCollection.test.js#L191-L193

Reply:
🙈